File: 백준/Bronze/10807. 개수 세기/개수 세기.py
import sys
fi = sys.stdin.readline
                
def solution():
    t = int(fi().rstrip())
    num = list(map(int, fi().split()))
    target = int(fi().rstrip())
    
    print(num.count(target))
    
if __name__ == '__main__':
    solution()

File: 백준/Bronze/10807. 개수 세기/개수 세기.java
import java.io.*;
import java.util.*;

public class Main {
    public static void solution() throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringBuilder sb = new StringBuilder();
        
        int n = Integer.parseInt(br.readLine());
        int[] arr = new int[201];
        StringTokenizer st = new StringTokenizer(br.readLine());
        for(int i=0;i<n;i++) {
            int num = Integer.parseInt(st.nextToken());
            arr[num+100]++;
        }
        int v = Integer.parseInt(br.readLine());
        sb.append(arr[v+100]);
        
        System.out.print(sb);
        br.close();
    }
    
    public static void main(String[] args) throws IOException {
        Main.solution();
    }
}